index.js

```diff
@@ -1 +1,2 @@
-alert('Hola usuario');
\ No newline at end of file
+const nombreUsuario = prompt('Ingrese su nombre')
+alert(`Hola ${nombreUsuario}`);
```

Comment:
En este caso me parece que es mas legible si usamos concatenacion.

```suggestion
alert(`Hola ${nombreUsuario}`);
```